docs/source/editor/prepub.rst

```diff
@@ -0,0 +1,99 @@
+It is important to ensure that the abstract and title in the metadata exactly match the abstract and title of the PDF file (often, authors change their abstract during revision and copyediting, so the original abstract needs updating before publication). 
+**The metadata abstract is displayed when you preview the XML file** (if you are using XML files) **in the Typesetting Plugin.**
+ 
+
+.. figure:: ../nstatic/gifprepub.gif
+
+If you copy and paste an abstract into the metadata, ensure you paste this in without formatting and, if necessary, use an HTML cleaning tool so that the abstract text matches the formatting of the rest of the XML text.
+**Use the preview XML file function in the Typesetting Plugin to check abstracts have updated correctly.**
+
+It is very important, for Open Access compliance purposes, that the licence selected in the metadata (which, by default, is CC BY-4.0) matches any licensing information given in the footer of the PDF file. 
+
+.. warning::
+   **If an author requests a licence other than CC BY-4.0, this must be flagged in advance to the typesetters.** 
+
+
+Once you have thoroughly checked the metadata, check the ‘Mark as Complete’ box to move onto the next step. 
+
+2. Set Issue
+------------
+
+Set Issue, the second step in Pre Publication, is where you can assign the article’s issue(s), if this has not been done already. 
+Click ‘Add to Issue’ to add to an existing issue (you can select more than one issue if needed, for example, if the article is part of a special issue as well as a journal volume) or ‘Create an issue’ at the bottom of the pop up window if the article belongs to a new issue. Then click the X at the top of the pop up box to close.
+
+Check the ‘Mark as Complete’ box to move onto the next step. 
+
+3. Verify DOIs
+--------------
+
+Articles usually have a DOI, or Digital Object Identifier, automatically assigned and ‘minted’ (created) for them by the system. This unique digital link to the article is usually created automatically for you by the system, and the Verify DOI stage lets you check if the DOI has been issued successfully or if there are any problems.
+If the DOI only has two ‘hops’, or two steps shown as bullet points in the Verify DOI stage, this means that no further changes are needed and you are ready to proceed. 
+You can verify that the article’s DOI is resolving correctly by clicking ‘Edit’ in the top right hand corner of the pop up window and checking that the ‘Status’ column reads ‘Success’. If the DOI Status is listed as pending, or blank, click ‘Poll for DOI Status’. If no DOI has been assigned, click ‘Issue DOI’. NB that newly issued DOIS can take a while to go through.  
+
+
+.. figure:: ../nstatic/prepubgif2.gif
+
+Check the ‘Mark as Complete’ box to move onto the next step. 
+
+4. Select a Galley for Rendering
+--------------------------------
+
+For OLH journals publishing on Janeway, always select the XML galley for rendering. 
```

Comment:
This is too OLH specific for readthedocs. Some publish HTML instead of XML and even publish Word. Perhaps say: you should select either an XML or HTML file (for OLH there will only be an XML option anyway so that is covered) and that if there are other files present that are not publishable galleys they can be removed...

Reply:
Will do!